validator/sawtooth_validator/journal/publisher.py

```diff
@@ -486,9 +486,7 @@ def __init__(self,
         self._transaction_executor = transaction_executor
         self._block_sender = block_sender
         self._batch_publisher = BatchPublisher(identity_signer, batch_sender)
-        self._pending_batches = []  # batches we are waiting for validation,
-        # arranged in the order of batches received.
```

Comment:
Typo in corresponding commit message: "using a three separate" ->  "using three separate"